### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by mast-eu**
## Proposed changes

- add badges for total downloads from SourceForge and Chocolatey (bfb7be53e2c6c11eaf94e9083b4ab2ef1be7c5a9)
- modify badges for latest downloads from GitHub and SourceForge; (087aa44f41d34b8a8ceab65b98ee2f1d5634b7a3)
- correct some links (to latest 2.x release, @pmiossec and @DmitryZhelnin user profiles) and minor text changes (ce5626b1d3cc4f4185ed62c28d8edb5f715bcc3e)
- reorder code for HTML tables. This brings no changes in content or layout, but improves readability.  (32b04b48b7f2512081634919273ed202b38f0395)

### Before

image

### After

image

## Test methodology

- manual

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Git Extensions
 
 Git Extensions is a standalone UI tool for managing git repositories.
-It also integrates with Windows Explorer and Microsoft Visual Studio (2015/2017).
+It also integrates with Windows Explorer and Microsoft Visual Studio (2015/2017/2019).
```
Comment (mast-eu) [inline README.md]: The same should probably be copied into the project description, too:
image

File: README.md [hunk 2]
```diff
@@ -59,75 +74,126 @@ The code is in maintenance mode with no significant active development planned.
...
+    <td>
+      <strong>v3.1</strong>
+    </td>
+    <td>
+      <a href="https://github.com/gitextensions/gitextensions/releases/latest">[ Download ]</a><br />
+      <a href="https://github.com/gitextensions/gitextensions/releases/latest" rel="nofollow" style="vertical-align: -webkit-baseline-middle;"><img src="https://img.shields.io/github/downloads/gitextensions/gitextensions/latest/total.svg?label=GitHub%20downloads%20(latest)&cacheSeconds=3600"></a> <a href="https://sourceforge.net/projects/gitextensions/" rel="nofollow" style="vertical-align: -webkit-baseline-middle;"><img src="https://img.shields.io/sourceforge/dt/gitextensions/v3.1.svg?label=SourceForge%20downloads%20(latest)&cacheSeconds=3600"></a>
```
Comment (mast-eu) [inline README.md]: Unfortunately, the badge `SourceForge doanloads (latest)` is not a true _latest_. It carries a version number and needs to be bumped with every release: `https://img.shields.io/sourceforge/dt/gitextensions/v3.1.svg`.

Comment (RussKie) [reply README.md]: oh well, there are few things like that.
we'll need to document them in the wiki - https://github.com/gitextensions/gitextensions/wiki/How-To%3A-prepare-a-release

File: README.md (hunk 1 above)
Comment (gerhardol) [reply README.md]: There is at least #6500 before VS2019 is fully supported....
README should be updated

Comment (mast-eu) [reply README.md]: As I understand, #6500 is for building GE itself with VS2019.
But is anything blocking the use of the current VSIX extension in VS2019?

**Comment by RussKie** [comment]
I've moved to vs2019 full-time,  no probs building or running

File: README.md (hunk 1 above)
Comment (gerhardol) [reply README.md]: No

File: README.md (hunk 2 above)
Comment (mast-eu) [reply README.md]: done

**Comment by gerhardol** [comment]
> I've moved to vs2019 full-time,  no probs building or running

Only when using build.cmd, it is there where #6500 is needed. MS likes to move around the build tools.

**Pull request merged**